From b155a5d7443048ad0544552a67cc2acd542ac077 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Thu, 11 Jun 2020 10:25:38 +0800
Subject: [PATCH] net: wireless: rockchip_wlan: rtl8822bs: Fix clang warning

drivers/net/wireless/rockchip_wlan/rtl8822bs/hal/phydm/phydm_dfs.c:1676:4:
warning: misleading indentation; statement is not part of the previous 'if'
[-Wmisleading-indentation]
                        dfs->pri_cond4 = 1;
                        ^

Signed-off-by: Tao Huang <huangtao@rock-chips.com>
Change-Id: Ieee0b2dac1d5fd6342bd7f6fd6a3e67af4c9b26c
---
 .../net/wireless/rockchip_wlan/rtl8822bs/hal/phydm/phydm_dfs.c  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/rockchip_wlan/rtl8822bs/hal/phydm/phydm_dfs.c b/drivers/net/wireless/rockchip_wlan/rtl8822bs/hal/phydm/phydm_dfs.c
index 6f1fbab7083e..08c7b6d7d478 100644
--- a/drivers/net/wireless/rockchip_wlan/rtl8822bs/hal/phydm/phydm_dfs.c
+++ b/drivers/net/wireless/rockchip_wlan/rtl8822bs/hal/phydm/phydm_dfs.c
@@ -1673,7 +1673,7 @@ void phydm_dfs_histogram_radar_distinguish(
 			dfs->pri_cond3 = 1;
 
 		/*@Cancel the condition that the abs between pri and pw*/
-			dfs->pri_cond4 = 1;
+		dfs->pri_cond4 = 1;
 
 		if (dfs->pri_hold_sum[5] <= dfs->pri_sum_g5_th)
 			dfs->pri_cond5 = 1;
-- 
2.35.3

